### PR TITLE
Add Scenery for Mac

### PR DESCRIPTION
**Pull request by dkhamsing**

### CODE REVIEW

File: Issues/Week72.md [hunk 1]
```diff
@@ -16,6 +16,7 @@
 * [QLImageset](https://github.com/qfish/QLImageset)
 * [ReactiveAnimation](https://github.com/ReactiveCocoa/ReactiveAnimation), by [@jspahrsummers](https://twitter.com/jspahrsummers)
 * [A swifter Core Data stack](https://github.com/jessesquires/JSQCoreDataKit), by [@jesse_squires](https://twitter.com/jesse_squires)
+* [Scenery for Mac](http://getscenery.com), by [@chriseidhof](https://twitter.com/chriseidhof)
```
Comment (RuiAAPeres) [inline Issues/Week72.md]: Shouldn't it be by `Unsigned Integer`?

**Comment by dkhamsing** [comment]
I think you're right
Let me update when I get back to my desk :smile:

**Comment by dkhamsing** [comment]
Updated

**Comment by RuiAAPeres** [comment]
:cake:

**Comment by dkhamsing** [comment]
:smile:

**Comment by dkhamsing** [comment]
cc @chriseidhof

**Comment by chriseidhof** [comment]
❤️

**Pull request merged**